### PR TITLE
Post-game

### PR DESCRIPTION
**Pull request by byronnill**

### CODE REVIEW

**Review by MaikaRenee — APPROVED** [review]

File: src/components/grid.vue [hunk 1]
```diff
@@ -146,7 +167,7 @@ export default {
   async mounted() {
     // Listen for board state changes
     gamesCollection
-      .doc('VUqGnWBLmgulz3X5O13h') // Obtain from state in the future when rooms are implemented
+      .doc(this.currentGame) // Obtain from state in the future when rooms are implemented
```
Comment (KenOafallasDLSU) [inline src/components/grid.vue:170]: anticipating many merge conflicts with the pregame on this file based on changes

File: src/components/chooseNewTimeModal.vue [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      this.toggle3 = this.toggle5 = this.toggle10 = false
+    },
+
+    /**
+     * updates the cloud firestore and the vuex state for the newly selected options for the rematch
+     * 
+     * @param lastPlayerMoved - user id of the player whose been set as the black player
+     * @param isHostWhite - boolean var indicating if the host will play as white
+     */
+    async updateDBAndState(lastPlayerMoved, isHostWhite) {
```
Comment (KenOafallasDLSU) [inline src/components/chooseNewTimeModal.vue:86]: local VueX stores will have to be updated too in the future

File: src/store/modules/cells/actions.js [hunk 3]
```diff
@@ -66,32 +66,32 @@ const actions = {
...
    * aKingCapturePiece moves a black or white king to capture an opposite-colored piece
    * in its diagonal
    * @param coords - an object containing the source and destination coordinates
    */
-  aKingCapturePiece({ commit, state }, payload) {
+  aKingCapturePiece({ commit, rootState, state }, payload) {
     const { coords, isPlayerBlack } = payload
     commit('mKingCapturePiece', coords)
-    writeBoardToDB(state.cells, isPlayerBlack)
+    writeBoardToDB(rootState.game.currentGameID, state.cells, isPlayerBlack)
```
Comment (KenOafallasDLSU) [inline src/store/modules/cells/actions.js:94]: action.js changes are completely redundant with changes for pregame, will anticipate the merge conflicts XD

**Review by KenOafallasDLSU — APPROVED** [review]
byron and I were basically doing the same stuff, so expecting a lot of merge conflicts in the future on the parts where we made the same features with slightly different implementations

**Pull request merged**